Buildings/Controls/OBC/ASHRAE/package.order

```diff
@@ -1 +1,2 @@
 G36_PR1
+PrimarySystem
```

Comment:
I'll merge 1378 branch into your base branch here so that this does not need to be added. Sorry about that.